css/main.css

```diff
@@ -292,19 +292,19 @@ body > div#index div#indexmenu {
 	position: fixed;
 	top: 0px;
 	left: -240px;
-	width: 220px;
+	width: 240px;
 	height: 100%;
-    margin: -10px 0px 0px 0px;
-	padding: 20px 10px 10px;
+	padding: 10px;
 	background-color: rgb(0, 146, 191);
 	color: #FFFFFF;
 	z-index: 1;
-	
 	-webkit-overflow-scrolling: touch;
 	overflow-y: auto;
-
 	-webkit-transition: -webkit-transform .4s;
 	transition: transform .4s;
+	box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	-moz-box-sizing: border-box;
```

Comment:
* why is the same thing twice? Does one suppose to be -webkit?
* Should never include the unprefixed before the prefixed. This will make the older implementation overwrite the new
* Probably don't need any prefixes at all as the browsers using the prefixes will have been auto-updated by now. Possible exception being -webkit-, but Android 4+ and iOS 5.1+ don't need the prefix and they're both pretty old.